src/components/Project.js

```diff
@@ -181,7 +181,7 @@ const Project = (props) => {
                 <p>{props.description}</p>
             </div>
             <div className="tags">
-                <span className="tag">{props.language}</span>
+                { props.language ? <span className="tag">{props.language}</span> : null}
```

Comment:
Do you think it would make sense to say the language for A-to-Z is `Markdown`?

Reply:
I think it makes sense although markdown is not a programming language.